README.md

```diff
@@ -1,3 +1,14 @@
 # youtube-homepage
 
-Basic youtube clone from the Odin Project
+Basic youtube clone project from the Odin's Project
+
+# Technologies
+- HTML <br>
+- CSS
+
+# Authors
+- <a href="https://github.com/sz16900">Seth Zea</a>
+- <a href="https://github.com/srueda27">Santiago Rueda Ortiz</a>
```

Comment:
Checkout the more correct syntax for writing links in markdown! You'll love it :smile:. See this [guid](https://guides.github.com/features/mastering-markdown/)

Reply:
Update the README file with the proper markdown sintax